libr/io/plugin.c

```diff
@@ -104,6 +104,9 @@ R_API int r_io_plugin_list_json(RIO *io) {
 	int n = 0;
 	io->cb_printf("{\"IO_Plugins\":[");
 	ls_foreach (io->plugins, iter, plugin) {
+		if(n != 0)
+			io->cb_printf(",");
```

Comment:
missing space before (

Reply:
i would rather prefer to put a %s in the line 115 (io->cb_printf ("%s{\"Permission... and pass n?",":"" as 2nd arg